crates/voicevox_core/src/publish.rs

```diff
@@ -12,10 +13,28 @@ use std::path::{Path, PathBuf};
 use std::sync::Mutex;
 
 // const PHONEME_LENGTH_MINIMAL: f32 = 0.01;
+const HIDDEN_SIZE: usize = 192;
 
 // static SPEAKER_ID_MAP: Lazy<BTreeMap<u32, (usize, u32)>> =
 //     Lazy::new(|| include!("include_speaker_id_map.rs").into_iter().collect());
 
+fn length_regulator(length: usize, embedded_vector: Vec<f32>, durations: &[f32]) -> Vec<f32> {
```

Comment:
こっちの形の方が一般的かと思います。

```suggestion
fn length_regulator(length: usize, embedded_vector: &[f32], durations: &[f32]) -> Vec<f32> {
```

(↓のsuggestionとセットで)

Reply:
ありがとうございます！適用します！